exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -106,6 +106,7 @@
   <p>STATE: ${model.getProfile().getState().name()}</p>
   <p>FOREMAN: ${model.getProfile().getForeman().getAddress()}</p>
   <p>TOTAL FRAGMENTS: ${model.getProfile().getTotalFragments()}</p>
+  <p>DURATION: ${model.getProfileDuration()}</p>
```

Comment:
The end time for a query is not set until the very end of the query, so the default value in 0L. For the `/profiles ` template, the elapsed time is computed using the current time in millis for running queries. Hence it shows correctly here. As per your suggestion, fixed it now by changing the signature to take the start and end times as arguments.